src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,25 +45,33 @@
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              Todo #{selectedTodo.id}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-            <button type="button" className="delete" data-cy="modal-close" />
+            <button
+              onClick={() => onTodoSelect(null)}
```

Comment:
Your method naming doesn't follow the conventions provided in the checklist. Consider renaming this method to something like `handleTodoSelect`.

Reply:
I just dont want to make a single props for using it once just for reseting...